src/site/data/Assignment1/.idea/modules.xml

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you still need these? Stuff in `.idea` contains info about how to build and run your project (like tests).

Reply:
I gave them the whole projects as a starting point for each, so they can open an "existing project". So it has src, test, resources all labelled.

Reply:
/shrug ?

Reply:
For now, yeah, let's go with that.

Nevertheless, it will be further looked into when assignment 1 is redone.